modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -762,6 +763,154 @@ public void testAddObjectEntry() throws Exception {
+							"textObjectFieldName", RandomTestUtil.randomString()
+						).build();
+					}
+				},
+				ObjectDefinitionConstants.SCOPE_COMPANY);
+
+		ObjectRelationshipTestUtil.relateObjectEntries(
+			parentObjectEntry2.getId(), childObjectEntry3.getId(),
+			objectRelationship, TestPropsValues.getUserId());
+
+		ObjectRelationshipTestUtil.relateObjectEntries(
+			parentObjectEntry2.getId(), childObjectEntry4.getId(),
+			objectRelationship, TestPropsValues.getUserId());
+
+		_assertEquals(
+			new ObjectEntry() {
+				{
+					properties = HashMapBuilder.<String, Object>put(
+						"averageAggregationObjectFieldName", "0"
+					).put(
+						"countAggregationObjectFieldName", "0"
+					).put(
+						"countAggregationObjectFieldNameSelf", "2"
+					).put(
+						"maxAggregationObjectFieldName", "0"
+					).put(
+						"minAggregationObjectFieldName", "0"
+					).put(
+						"sumAggregationObjectFieldName", "0"
+					).put(
+						"textObjectFieldName",
+						MapUtil.getString(
+							parentObjectEntry2.getProperties(),
+							"textObjectFieldName")
+					).build();
+				}
+			},
+			_defaultObjectEntryManager.getObjectEntry(
+				_simpleDTOConverterContext, _objectDefinition1,
+				parentObjectEntry2.getId()));
+
+		_defaultObjectEntryManager.deleteObjectEntry(
+			_objectDefinition1, childObjectEntry3.getId());
+
+		_assertEquals(
+			new ObjectEntry() {
+				{
+					properties = HashMapBuilder.<String, Object>put(
+						"averageAggregationObjectFieldName", "0"
+					).put(
+						"countAggregationObjectFieldName", "0"
+					).put(
+						"countAggregationObjectFieldNameSelf", "1"
+					).put(
+						"maxAggregationObjectFieldName", "0"
+					).put(
+						"minAggregationObjectFieldName", "0"
+					).put(
+						"sumAggregationObjectFieldName", "0"
+					).put(
+						"textObjectFieldName",
+						MapUtil.getString(
+							parentObjectEntry2.getProperties(),
+							"textObjectFieldName")
+					).build();
+				}
+			},
+			_defaultObjectEntryManager.getObjectEntry(
+				_simpleDTOConverterContext, _objectDefinition1,
+				parentObjectEntry2.getId()));
+
+		_defaultObjectEntryManager.deleteObjectEntry(
+			_objectDefinition1, childObjectEntry4.getId());
+
+		_assertEquals(
+			new ObjectEntry() {
+				{
+					properties = HashMapBuilder.<String, Object>put(
+						"averageAggregationObjectFieldName", "0"
+					).put(
+						"countAggregationObjectFieldName", "0"
+					).put(
+						"countAggregationObjectFieldNameSelf", "0"
+					).put(
+						"maxAggregationObjectFieldName", "0"
+					).put(
+						"minAggregationObjectFieldName", "0"
+					).put(
+						"sumAggregationObjectFieldName", "0"
+					).put(
+						"textObjectFieldName",
+						MapUtil.getString(
+							parentObjectEntry2.getProperties(),
+							"textObjectFieldName")
+					).build();
+				}
+			},
+			_defaultObjectEntryManager.getObjectEntry(
+				_simpleDTOConverterContext, _objectDefinition1,
+				parentObjectEntry2.getId()));
```

Comment:
@pedro-oliveira446,

I believe you don't need to define the entire DTO because you're only testing the countAggregationObjectFieldNameSelf:

try this:

```java
new ObjectEntry() {
		{
			properties = HashMapBuilder.<String, Object>put(
				"countAggregationObjectFieldNameSelf", "0"
			).build();
		}
	}
```

Thanks!